packages/wpcom-template-parts/src/universal-header-navigation/menu-items.tsx

```diff
@@ -8,6 +9,48 @@ export const NonClickableItem = ( { content, className }: MenuItemProps ) => {
 	);
 };
 
+const getParentElement = ( node: HTMLElement | null, pattern: RegExp ) => {
+	let parent = node;
+	while ( parent && ! parent.className.match( pattern ) ) {
+		if ( parent === document.body ) {
+			return null;
+		}
+		parent = parent.parentElement;
+	}
+
+	return parent;
+};
+
+const clickNavLinkEvent = ( target: HTMLElement ) => {
+	const props: { [ key: string ]: string | number } = {};
+
+	const container = getParentElement( target, /container/ );
+	const section = getParentElement( target, /section/ );
```

Comment:
Not really an issue, but if the element has two children with the section class name then this might stop at the first one, but then once that is found, it should suffice (per my understanding)

Reply:
Sounds correct.
I think that getting the first-found element is enough for now.
We can follow up with adjustments if needed.